packages/apps/dashboard/ui-2024/src/pages/SearchResults/WalletAddress/WalletAddress.tsx

```diff
@@ -67,7 +67,12 @@ const WalletAddress = ({
 					<TitleSectionWrapper title="Balance">
 						<Stack sx={{ whiteSpace: 'nowrap', flexDirection: 'row' }}>
 							<Typography variant="body2">
-								<FormatNumberWithDecimals value={balance} />
+								<NumericFormat
+									displayType="text"
+									value={Number(balance) < 1 ? Number(balance) * 1e18 : balance}
```

Comment:
We should show the decimals of the balance also 
![image](https://github.com/user-attachments/assets/c12de41a-13c1-43b6-b480-8f9d04bfacbd)

Reply:
Done 👍